src/Posts/Feed.jsx

```diff
@@ -34,6 +34,14 @@ const renderItem = (item) => {
   );
 };
 
+if (posts.length === 0) {
```

Comment:
As from now we are more specific with filtering posts I decided to add this placeholder for users who doesn't have any posts from follow accounts